### DISCUSSION TITLE
Facebook login integration

### DISCUSSION THREAD

**Discussion by jbrandelero** [unoplatform/uno#4965] [Q&A]
Hi, is facebook login integration possible? Theres any example?

**Comment by jeromelaban** [comment]
Thanks for your interest in the Uno Platform! 

The support for facebook can be done through the standard APIs, and standard authentication techniques with the WebView control should be enough, using the `Xamarin.Facebook.Android` and `Xamarin.Facebook.iOS` packages. For WebAssembly, the recipe has yet to be defined.

**Reply by trydalch** [reply]
I realize this is old, but I couldn't find anything newer than this on FB login. 

It would be nice to have some documentation on how to use Xamarin (.net for ios/android) packages. Also, some direction about the sorts of changes we can make inside the `Platforms/*` folders. 

I've looked at using Xamarin.Facebook.iOS, but their samples reference AppDeletgates and such, and it's not clear to me where/how to do that in the context of Uno single-project apps.

**Comment by egarim** [comment]
Is there a recipe for webassembly?

**Comment by jeromelaban** [comment]
This space has been improving recently with the integration of MSAL.NET. There's an in-progress PR on the topic: https://github.com/unoplatform/uno/pull/3262

/cc @carldebilly

**Reply by francoistanguay** [reply]
Isnt the answer for us to offer guidance on OIDC? https://github.com/unoplatform/uno/issues/3393
